flow/core/kernel/kernel.py

```diff
@@ -99,10 +100,21 @@ def update(self, reset):
         """
         self.vehicle.update(reset)
         self.traffic_light.update(reset)
-        self.scenario.update(reset)
+        self.network.update(reset)
         self.simulation.update(reset)
 
     def close(self):
-        """Terminate all components within the simulation and scenario."""
-        self.scenario.close()
+        """Terminate all components within the simulation and network."""
+        self.network.close()
         self.simulation.close()
+
+    @property
+    def scenario(self):
```

Comment:
shouldn't this be called `def network(self)`?

Reply:
I'm adding deprecated version of scenario wherever it has been used in the past. We can leave them around until at least version 0.6.0